hardware/raspberrypi/boot_diagnostics.md

```diff
@@ -3,9 +3,7 @@
 The bootloader on the Raspberry Pi 4 is stored in EEPROM, and as well as providing boot services, is also capable of providing diagnostic information on an attached HDMI display. To display this information, remove the SD card from the Raspberry Pi 4, and reboot. A diagnostic display similar to below should appear on the attached display.
 
 This diagnostics page will also appear if the bootloader is unable to boot from an inserted SD card, or is unable to network boot; for example, if there is no bootable image on the card, or it is defective, or the network boot parameters are incorrect.
-
-
-
+A power cycle is required in order to trigger a reboot.
```

Comment:
What does that sentence mean? Does it mean simply that you have to reboot the Pi to see the diagnostics screen?

Reply:
I've updated the text to make it clearer.